df_py/util/dftool_module.py

```diff
@@ -166,9 +179,9 @@ def do_volsym():
     (Vi, Ci, SYMi) = retryFunction(
         queries.queryVolsOwnersSymbols, RETRIES, 60, rng, CHAINID
     )
-    csvs.saveNftvolsCsv(Vi, CSV_DIR, CHAINID)
-    csvs.saveOwnersCsv(Ci, CSV_DIR, CHAINID)
-    csvs.saveSymbolsCsv(SYMi, CSV_DIR, CHAINID)
+    csvs.save_nftvols_csv(Vi, CSV_DIR, CHAINID)
```

Comment:
these name changes will create conflicts with the args branch, but we'll sort it out. I think they are beneficial, so let's keep them.